packages/node/src/handlers/call-api.ts

```diff
@@ -1,27 +1,59 @@
 import * as adapter from '@api3/adapter';
-import { OIS, SecurityScheme } from '@api3/ois';
-import { go, retryOnTimeout } from '../utils/promise-utils';
-import { removeKeys } from '../utils/object-utils';
+import { OIS, ReservedParameterName, SecurityScheme } from '@api3/ois';
+import { getResponseParameters, RESERVED_PARAMETERS } from '../adapters/http/parameters';
 import { getEnvValue } from '../config';
+import { API_CALL_TIMEOUT, API_CALL_TOTAL_TIMEOUT } from '../constants';
 import * as logger from '../logger';
-import { getResponseParameters, RESERVED_PARAMETERS } from '../adapters/http/parameters';
 import {
   AggregatedApiCall,
+  ApiCallParameters,
   ApiCallResponse,
+  ChainConfig,
   Config,
   LogsData,
   RequestErrorCode,
   SecuritySchemeEnvironmentConfig,
 } from '../types';
-import { API_CALL_TIMEOUT, API_CALL_TOTAL_TIMEOUT } from '../constants';
+import { removeKeys } from '../utils/object-utils';
+import { go, retryOnTimeout } from '../utils/promise-utils';
+
+function addMetadataParameters(
+  parameters: ApiCallParameters,
+  aggregatedApiCall: AggregatedApiCall,
+  chain: ChainConfig
+) {
+  switch (aggregatedApiCall.parameters[ReservedParameterName.RelayMetadata]) {
+    case 'v1':
+      return {
+        ...parameters,
+        _airnode_airnode_id: aggregatedApiCall.airnodeId,
+        _airnode_client_address: aggregatedApiCall.clientAddress,
+        _airnode_designated_wallet: aggregatedApiCall.designatedWallet,
+        _airnode_endpoint_id: aggregatedApiCall.endpointId,
+        _airnode_requester_index: aggregatedApiCall.requesterIndex,
+        _airnode_request_id: aggregatedApiCall.id,
+        _airnode_chain_type: aggregatedApiCall.chainId,
+        _airnode_chain_id: chain.type,
+        _airnode_airnode_rrp: chain.contracts.AirnodeRrp,
+      };
+    default:
+      return parameters;
+  }
+}
 
 function buildOptions(
+  chain: ChainConfig,
   ois: OIS,
   securitySchemeEnvironmentConfigs: SecuritySchemeEnvironmentConfig[],
   aggregatedApiCall: AggregatedApiCall
 ): adapter.BuildRequestOptions {
+  const parameters = aggregatedApiCall.parameters;
+
+  // Include airnode metadata based on _relay_metadata version number
+  const parametersWithMetadata = addMetadataParameters(parameters, aggregatedApiCall, chain);
```

Comment:
Why pass in `parameters` and `aggregatedApiCall` separately? We can access parameters in `addMetadataParameters()` through the `aggregatedApiCall`?